### ISSUE TITLE
Doesn't work with Mac OSX

### ISSUE THREAD

**Issue by telenieko** [mweagle/Sparta#101]
Documentation does not state which platforms are supported for development, so one would guess any platform supported by golang is good but...

Following the getting started from Mac OS, when doing the go run main.go ...
```
INFO[0000] ../../github.com/mweagle/Sparta/sparta_main_awsbinary.go:17:2: cannot find package "github.com/zcalusic/sysinfo" in any of:

INFO[0000] 	/Users/marc/.gvm/gos/go1.9/src/github.com/zcalusic/sysinfo (from $GOROOT)
INFO[0000] 	/Users/marc/go/src/github.com/zcalusic/sysinfo (from $GOPATH)
INFO[0000] Invoking rollback functions
```

As far as I understand zcalusic/sysinfo is Linux only.

> go version go1.9 darwin/amd64
GOARCH="amd64"
GOBIN="/Users/marc/go/bin"
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
CC="clang"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/lc/80799yss6klb3_ft_szyfgyr0000gn/T/go-build360770781=/tmp/go-build -gno-record-gcc-switches -fno-common"
CXX="clang++"

**Comment by mweagle** [comment]
Hi @telenieko - fair assumption & I do develop on OSX. I suspect the build tags may be interfering with `go get`s ability to fetch the package. Will see if there's a workaround. In the meantime, if you execute:
```
go get github.com/zcalusic/sysinfo
```
directly are you able to build a Sparta app?

**Comment by mweagle** [comment]
Added a precondition check - will be available in 1.0.2:

```
...

ERRO[0000] Failed to validate preconditions: Please run `go get -u -v github.com/zcalusic/sysinfo` to install this Linux-only package. This package is used when cross-compiling your AWS Lambda binary and cannot be reliably imported across platforms
exit status 1
```

**Comment by mweagle** [comment]
Updated error text:

**Comment by mweagle** [comment]
Released with [1.1.0](https://github.com/mweagle/Sparta/blob/master/CHANGES.md). Closing.
